torch/testing/_internal/opinfo/definitions/signal.py

```diff
@@ -0,0 +1,248 @@
+            DecorateInfo(
+                unittest.skip("Skipped!"), "TestCommon", "test_noncontiguous_samples"
+            ),
+            DecorateInfo(
+                unittest.skip("Skipped!"),
+                "TestCommon",
+                "test_variant_consistency_eager",
+            ),
+            DecorateInfo(unittest.skip("Skipped!"), "TestMathBits", "test_conj_view"),
+            DecorateInfo(
+                unittest.skip("Skipped!"), "TestMathBits", "test_neg_conj_view"
+            ),
+            DecorateInfo(unittest.skip("Skipped!"), "TestMathBits", "test_neg_view"),
+            # UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
+            DecorateInfo(unittest.expectedFailure, "TestCommon", "test_out_warning"),
+        ),
+    ),
+    OpInfo(
+        "signal.windows.gaussian",
+        ref=scipy.signal.get_window if TEST_SCIPY else None,
+        dtypes=all_types_and(torch.float, torch.double, torch.long),
+        dtypesIfCUDA=all_types_and(
+            torch.float, torch.double, torch.bfloat16, torch.half, torch.long
+        ),
+        sample_inputs_func=sample_inputs_window,
+        error_inputs_func=error_inputs_gaussian_window,
+        supports_out=False,
+        supports_autograd=False,
+        skips=(
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestNormalizeOperators",
+                "test_normalize_operator_exhaustive",
+            ),
+            # TODO: same as this?
+            # https://github.com/pytorch/pytorch/issues/81774
+            # also see: arange, new_full
+            # fails to match any schemas despite working in the interpreter
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestOperatorSignatures",
+                "test_get_torch_func_signature_exhaustive",
+            ),
+            # fails to match any schemas despite working in the interpreter
+            DecorateInfo(
+                unittest.expectedFailure, "TestJit", "test_variant_consistency_jit"
+            ),
+            # skip these tests since we have non tensor input
+            DecorateInfo(
+                unittest.skip("Skipped!"), "TestCommon", "test_noncontiguous_samples"
+            ),
+            DecorateInfo(
+                unittest.skip("Skipped!"),
+                "TestCommon",
+                "test_variant_consistency_eager",
+            ),
+            DecorateInfo(unittest.skip("Skipped!"), "TestMathBits", "test_conj_view"),
+            DecorateInfo(
+                unittest.skip("Skipped!"), "TestMathBits", "test_neg_conj_view"
+            ),
+            DecorateInfo(unittest.skip("Skipped!"), "TestMathBits", "test_neg_view"),
+            # UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
+            DecorateInfo(unittest.expectedFailure, "TestCommon", "test_out_warning"),
```

Comment:
Weird that this one is even triggered when `supports_out=True`.

Reply:
I'll look into it before wrapping up the PR.

Reply:
We don't need to xfail or skip it. It succeeds for this OpInfos. I removed it.